src/Symfony/Component/AssetMapper/Tests/ImportMap/ImportMapUpdateCheckerTest.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Tests\ImportMap;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\AssetMapper\ImportMap\ImportMapConfigReader;
+use Symfony\Component\AssetMapper\ImportMap\ImportMapEntries;
+use Symfony\Component\AssetMapper\ImportMap\ImportMapEntry;
+use Symfony\Component\AssetMapper\ImportMap\ImportMapType;
+use Symfony\Component\AssetMapper\ImportMap\ImportMapUpdateChecker;
+use Symfony\Component\AssetMapper\ImportMap\PackageUpdateInfo;
+use Symfony\Component\HttpClient\MockHttpClient;
+use Symfony\Component\HttpClient\Response\MockResponse;
+
+class ImportMapUpdateCheckerTest extends TestCase
+{
+    private ImportMapConfigReader $importMapConfigReader;
+    private ImportMapUpdateChecker $updateChecker;
+
+    protected function setUp(): void
+    {
+        $this->importMapConfigReader = $this->createMock(ImportMapConfigReader::class);
+        $httpClient = new MockHttpClient();
+        $httpClient->setResponseFactory(self::responseFactory(...));
+        $this->updateChecker = new ImportMapUpdateChecker($this->importMapConfigReader, $httpClient);
+    }
+
+    public function testGetAvailableUpdates()
+    {
+        $this->importMapConfigReader->method('getEntries')->willReturn(new ImportMapEntries([
+            '@hotwired/stimulus' => new ImportMapEntry(
+                importName: '@hotwired/stimulus',
+                version: '3.2.1',
+                packageName: '@hotwired/stimulus',
+            ),
+            'json5' => new ImportMapEntry(
+                importName: 'json5',
+                version: '1.0.0',
+                packageName: 'json5',
+            ),
+            'bootstrap' => new ImportMapEntry(
+                importName: 'bootstrap',
+                version: '5.3.1',
+                packageName: 'bootstrap',
+            ),
+            // Css will always be considered up to date
```

Comment:
Why CSS should be always considered up-to-date? At L95 it's expected to have `updateType: 'patch'`

Reply:
Nice catch, that's a comment that should have been deleted when I made the change to allow css to be managed. Thanks